components/inputs/input.css

```diff
@@ -0,0 +1,86 @@
+input{
+    outline:none;
+    border:none;
+    background-color:var(--bg-color);
+    position:relative;
+}
+
+.input-box{
+    
+    border:1px solid var(--text-color);
+    padding:1rem;
+    border-radius:5px;
+    width:100%;
+    font-size:1.6rem;
+}
+.input-box:focus{
+    border:1.5px solid var(--primary-color);
+}
+/******************/
+.input-s{
+    padding:0.5rem;
+    font-size:1.2rem;
+}
+.input-md{
+    padding:1rem;
+    font-size:1.6rem;
+}
+.input-lg{
+padding:1.8rem;
+font-size:2rem;
+}
+/*************************/
+.all-border-input{
+    border:1px solid var(--text-color);
+    border-radius:5px;
+    width:100%;
+}
+.all-border-input:focus{
+    border:2px solid var(--primary-color);
+}
+.bottom-border-input{
+    border-bottom:1px solid var(--text-color);
+    width:100%;
+}
+.bottom-border-input:focus{
+    border-bottom:2px solid var(--primary-color);
+}
+.no-border-input{
+    border:none;
+}
+/******************/
+.valid-input{
+    background-color: rgb(132, 241, 201,0.5);
+    color:#059862;
+    border: 1px solid #059862;
```

Comment:
The color code used here can be added in root to avoid repetition and it helps to maintain the code.

Reply:
yes sure!